src/jest.ml

```diff
@@ -167,7 +168,7 @@ module Runner (A : Asserter) = struct
         finish ());
         Js.undefined
       )
-      (Js.Undefined.from_opt timeout)
+    (Js.Undefined.from_opt (Js.Option.map ((fun v -> float_of_int v)[@bs ]) timeout))
```

Comment:
You don't need `float_of_int` here. From a JS point-of-view, a `float` and an `int` are exactly the same, they're both just `number`s, and `float_of_int` does nothing other than change the type the compiler sees.

Reply:
Thanks. Fixed.